mdlinkcheckconfig.json

```diff
@@ -28,6 +28,20 @@
       "reason": "500"
     },
     {
+
+      "pattern": "https://www.esa.int/",
+      "reason": "403"
+    },
```

Comment:
Remove this block. You don't add this link, so it shouldn't be added in this PR.